terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java

```diff
@@ -0,0 +1,203 @@
+        assertThat(form.getField("paymentMethod"), is("Bank Transfer"));
+        assertThat(form.getField("paymentAccount"), is(
+                "Current Account：12345678"));
+        assertThat(form.getField("childCount"), is("3"));
+        assertThat(form.getField("tourName"), is("【おすすめ】Terasolunaツアー(6泊7日)"));
+        assertThat(form.getField("accomName"), is("TERASOLUNAホテル第一荘"));
+        assertThat(form.getField("customerKana"), is("キムラ　タロウ"));
+        assertThat(form.getField("customerTel"), is("111-1111-1111"));
+        assertThat(form.getField("adultUnitPrice"), is("75000"));
+        assertThat(form.getField("reservedDay"), is("2019/02/21"));
+        assertThat(form.getField("conductor"), is("Yes"));
+        assertThat(form.getField("tourAbs"), is(
+                "そこは別天地、静寂と湯けむりの待つ宿へ… 詳しい情報はお取り合わせをお願い致します。"));
+        assertThat(form.getField("customerAdd"), is("千葉県八千代市上高野"));
+        assertThat(form.getField("customerJob"), is("プログラマ"));
+        assertThat(form.getField("tourDays"), is("7"));
+        assertThat(form.getField("depDay"), is("2019/03/31"));
+        assertThat(form.getField("customerName"), is("木村　太郎"));
+        assertThat(form.getField("childUnitPrice"), is("37500"));
+        assertThat(form.getField("depName"), is("北海道"));
+        assertThat(form.getField("customerBirth"), is("1975/01/05"));
+        assertThat(form.getField("arrName"), is("北海道"));
+        assertThat(form.getField("customerMail"), is("tarou@example.com"));
+        assertThat(form.getField("adultCount"), is("5"));
+        assertThat(form.getField("customerCode"), is("00000001"));
+        assertThat(form.getField("reserveNo"), is("00000001"));
+        assertThat(form.getField("remarks"), is("特になし"));
+        assertThat(form.getField("accomTel"), is("018-123-4567"));
+        assertThat(form.getField("customerPost"), is("276-0022"));
+        assertThat(form.getField("printDay"), is("2019/03/06"));
+        assertThat(form.getField("adultPrice"), is("375000"));
+        assertThat(form.getField("childPrice"), is("112500"));
+        assertThat(form.getField("sumPrice"), is("487500"));
+        assertThat(form.getField("paymentTimeLimit"), is("2019/03/24"));
+        assertThat(form.getFields().size(), is(35));
+
+        assertThat(response.getHeader("Content-Disposition"), is(
+                "attachment; filename=reservationReport.pdf"));
+    }
+
+    @Test
+    public void testPrepareResponse() {
+
+        HttpServletRequest request = new MockHttpServletRequest();
+
+        HttpServletResponse response = new MockHttpServletResponse();
+
+        reservationReportPdfStamperView.prepareResponse(request, response);
+        assertThat(response.getHeader("Pragma"), is("private"));
+        assertThat(response.getHeader("Cache-Control"), is(
+                "private, must-revalidate"));
+        assertThat(response.getCharacterEncoding(), is(StandardCharsets.UTF_8
+                .name()));
```

Comment:
I recmmend to add comment to explain `CharacterEncoding` is responsibility of this class , and Cache-Control Headers are responsibility of super classes.

Reply:
54e971965e11d5a723d864d7a1b7cd72631da35b

I add comment in above commit .